src/content/protocols/stakewise-v3/ethereum.md

```diff
@@ -0,0 +1,250 @@
+
+In addition to full logic upgrades, a large number of parameter changes can be performed without modifying bytecode. These parameters include fee destinations, blocklist administration, validator roots, keeper thresholds, reward delays, liquidation settings, vault capacities, the controller of OsToken, and the set of redeemable positions in the redeemer. Because these parameters influence how assets flow through the system, how user positions evolve over time and how rewards are computed or distributed, changes made through these permissions can materially alter the behaviour users experience, even if the underlying logic contracts remain the same.
+
+The upgrade keys are held by a small set of actors, mainly the StakeWise DAO multisig (controlling OsToken, VaultController, VaultsRegistry, and Keeper), and Stakewise Multisig #1 (controlling the Redeemer, MerkleDistributor, and MetaVaultFactory). Additionally, a dedicated 2-of-4 Blocklist Manager handles daily censorship permissions, though its authority is granted by Stakewise Multisig #2. These roles are able to enact upgrades or parameter changes without enforced delays at the protocol layer. As a result, the entities controlling these keys can, at any moment, replace logic contracts or adjust parameters affecting how vault shares are minted or burned, how assets are redeemed, how validators are assigned, how blocklists are managed and how rewards are distributed.
+
+These permissions mean that upgrades can directly influence both the handling of user funds and the treatment of unclaimed staking rewards. Logic upgrades to vaults or controllers can modify withdrawal or redemption behaviour. Changes to validator roots or key managers can reassign the staking infrastructure associated with user deposits. Adjustments to liquidation or redemption configuration can affect how OsToken positions settle. Administrative control over fee settings, treasury recipients and reward distribution parameters can alter how yield flows through the system. In multiple cases, an incorrect or hostile upgrade would allow the responsible administrator to redirect assets, obstruct withdrawals or modify redemption terms in ways that affect users’ positions.
+
+> Upgradeability score: High
+
+## Autonomy
+
+StakeWise v3 operates through several external and internal dependencies that influence its behaviour and determine how user funds and rewards are handled. These dependencies include the validator infrastructure managed by privileged entities, the oracle system feeding validator and reward data into the Keeper, and the multi-sig actors controlling key components such as vault implementations, OsToken parameters, the Redeemer, the MerkleDistributor and registry contracts. Because these dependencies provide inputs that the protocol relies on to recognise staking rewards, assign validators, process redemptions and compute vault accounting, their actions directly shape the system’s ongoing operation.
```

Comment:
The multisig signers are not external dependencies, they're part of the protocol